main/settings.js

```diff
@@ -31,6 +31,19 @@ async function setup (ctx) {
       clipboard.writeText(await wallet.getSeedPhrase())
     }
   }
+
+  ctx.importSeedPhrase = async () => {
+    const button = showDialogSync({
+      title: 'Import Seed Phrase',
+      // eslint-disable-next-line max-len
+      message: 'The seed phrase is used in order to back up your wallet, or move it to a different machine. Please copy it to your clipboard before proceeding. Please be cautious, as this will overwrite the seed phrase currently used, which will be permanently lost.',
```

Comment:
I'd like @patrickwoodhead to review this message.

Do we need to explain what is the seed phrase? I would expect the user to understand that if they decide to import the seed phrase,

I think it's also best to put the call to action as the last sentence in the message.

Should we also tell the user that they may want to export the current seed phrase?

```suggestion
      message: 'Please be cautious. This action will overwrite the seed phrase currently used, which will be permanently lost. Please copy the seed phrase to import to your clipboard before proceeding.',
```

Reply:
For reference, we're also not explaining what a seed phrase is in the export section.

Reply:
In your suggestion it says `Please copy the seed phrase to import`, but I think it needs to be `Please copye the seed phrase to your clipboard, to import`

Reply:
+1 to moving the CTA to the end

Reply:
Yeah I agree that we need to ask the user to be cautious as the first thing. Also, I'm a bit unclear what the CTA is asking the user to do. Is it asking them to copy their current seed phrase out, or copy their new one into the clipboard so that they can paste it into the input box?

Reply:
The user needs to have their current seed phrase copied in their clipboard, from which we will then import it